tests/CMakeLists.txt

```diff
@@ -15,100 +15,65 @@
-
-    add_definitions(-DVK_USE_PLATFORM_WIN32_KHR -DWIN32_LEAN_AND_MEAN)
-    # Workaround for TR1 deprecation in Visual Studio 15.5 until Google Test is updated
-    add_definitions(-D_SILENCE_TR1_NAMESPACE_DEPRECATION_WARNING)
-elseif(ANDROID)
-    add_definitions(-DVK_USE_PLATFORM_ANDROID_KHR)
-elseif(APPLE)
-    add_definitions(-DVK_USE_PLATFORM_MACOS_MVK)
-elseif(UNIX AND NOT APPLE) # i.e.: Linux
-    target_compile_options(vk_loader_validation_tests PUBLIC -fPIC)
-    if(BUILD_WSI_XCB_SUPPORT)
-        add_definitions(-DVK_USE_PLATFORM_XCB_KHR)
-    endif()
+# if(WIN32)
+#     target_compile_options(vk_loader_validation_tests PUBLIC ${MSVC_LOADER_COMPILE_OPTIONS})
 
-    if(BUILD_WSI_XLIB_SUPPORT)
-        add_definitions(-DVK_USE_PLATFORM_XLIB_KHR)
-    endif()
+#     add_definitions(-DVK_USE_PLATFORM_WIN32_KHR -DWIN32_LEAN_AND_MEAN)
+#     # Workaround for TR1 deprecation in Visual Studio 15.5 until Google Test is updated
+#     add_definitions(-D_SILENCE_TR1_NAMESPACE_DEPRECATION_WARNING)
+# elseif(ANDROID)
+#     add_definitions(-DVK_USE_PLATFORM_ANDROID_KHR)
+# elseif(APPLE)
+#     add_definitions(-DVK_USE_PLATFORM_MACOS_MVK)
+# elseif(UNIX AND NOT APPLE) # i.e.: Linux
+#     target_compile_options(vk_loader_validation_tests PUBLIC -fPIC)
+#     if(BUILD_WSI_XCB_SUPPORT)
+#         add_definitions(-DVK_USE_PLATFORM_XCB_KHR)
+#     endif()
 
-    if(BUILD_WSI_WAYLAND_SUPPORT)
-        add_definitions(-DVK_USE_PLATFORM_WAYLAND_KHR)
-    endif()
+#     if(BUILD_WSI_XLIB_SUPPORT)
+#         add_definitions(-DVK_USE_PLATFORM_XLIB_KHR)
+#     endif()
 
-    if(BUILD_WSI_SCREEN_QNX_SUPPORT)
-        add_definitions(-DVK_USE_PLATFORM_SCREEN_QNX)
-    endif()
-else()
-    message(FATAL_ERROR "Unsupported Platform!")
-endif()
+#     if(BUILD_WSI_WAYLAND_SUPPORT)
+#         add_definitions(-DVK_USE_PLATFORM_WAYLAND_KHR)
+#     endif()
 
-if(WIN32)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_CRT_SECURE_NO_WARNINGS -D_USE_MATH_DEFINES")
+#     if(BUILD_WSI_SCREEN_QNX_SUPPORT)
+#         add_definitions(-DVK_USE_PLATFORM_SCREEN_QNX)
+#     endif()
+# else()
+#     message(FATAL_ERROR "Unsupported Platform!")
+# endif()
 
-    # If MSVC, disable some signed/unsigned mismatch warnings.
-    if(MSVC)
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4267")
-    endif()
+# if(WIN32)
+#     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_CRT_SECURE_NO_WARNINGS -D_USE_MATH_DEFINES")
 
-else()
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-endif()
+#     # If MSVC, disable some signed/unsigned mismatch warnings.
+#     if(MSVC)
+#         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4267")
+#     endif()
+
+# else()
+#     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+# endif()
+
+# include_directories(${PROJECT_SOURCE_DIR}/external
+#                     ${GTEST_SOURCE_DIR}/googletest/include
+#                     ${CMAKE_CURRENT_BINARY_DIR}
+#                     ${CMAKE_BINARY_DIR}
+#                     ${PROJECT_BINARY_DIR})
```

Comment:
NIT: Why not remove?

Reply:
That was a mistake, should be fixed now.